src/api/src/application/Yoma.Core.Api/Startup.cs

```diff
@@ -128,24 +131,31 @@ public void Configure(IApplicationBuilder app)
       if (_appSettings.HttpsRedirectionEnabledEnvironmentsAsEnum.HasFlag(_environment)) app.UseHttpsRedirection();
       app.UseStaticFiles();
       app.UseRouting();
+      app.UseAuthentication();
+      app.UseAuthorization();
+      app.UseEndpoints(endpoints =>
+      {
+        // basic check to ensure the API is up
+        endpoints.MapHealthChecks("/api/v3/health/ready", new HealthCheckOptions
+        {
+          Predicate = (check) => check.Tags.Contains("ready")
+        }).AllowAnonymous();
+
+        // more detailed check to ensure the API can connect to the database
+        endpoints.MapHealthChecks("/api/v3/health/live", new HealthCheckOptions
+        {
+          Predicate = (check) => check.Tags.Contains("live")
+        }).AllowAnonymous();
+      });
 
       //enabling sentry tracing causes endless information logs about 'Sentry trace header is null'
       //if (_environment != Domain.Core.Environment.Local) app.UseSentryTracing();
-
-      app.UseAuthentication();
-      app.UseAuthorization();
       #endregion
 
       #region 3rd Party
       app.UseHangfireDashboard(options: new DashboardOptions
       {
         DarkModeEnabled = true,
-        /*
-        TODO: Resolve shared Data Protection Keys (NFS or S3 or SSM or Postgres) and set this back to `false`
-        * https://learn.microsoft.com/en-us/aspnet/core/security/data-protection/implementation/key-storage-providers
-        * https://learn.microsoft.com/en-us/aspnet/core/security/data-protection/configuration/overview
-        * https://github.com/aws/aws-ssm-data-protection-provider-for-aspnet
-        */
         IgnoreAntiforgeryToken = true, //replicas >=2 will cause antiforgery token issues
```

Comment:
I think, now that we have the Data Protection Keys in Redis, we can set this to `false`
```suggestion
        IgnoreAntiforgeryToken = false,
```

Reply:
We can always do this in another ticket/PR

Reply:
Ok, will do. Want to quickly centralize the api version in the route. Will include this in the PR